frontend/src/app/modules/user-profile/select-topics-page/select-topics-page.component.sass

```diff
@@ -0,0 +1,103 @@
+@import '../../../../styles/colors'
+
+@font-face
+    font-family: Barlow
+    src: url('../../../../assets/fonts/Barlow-Medium.ttf')
+
+main
+    width: 50%
+    margin: 30px auto 0 auto
+    font-family: Barlow, serif
+
+#back
+    background-color: $white
+    padding: 10px
+    border: none
+    width: 15%
+    display: flex
+    justify-content: center
+    align-items: center
+
+#back:hover, button:hover
+    background-color: rgba(0,0,0, 0.2)
+    cursor: pointer
+    transition: background-color 0.3s ease-out
+
+#back img
+    padding: 0 10px 0 0
+
+h2
+    color: $main-color
+    margin: 10px 0 0 0
+
+#topics
+    display: grid
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr
+    grid-gap: 30px
+    margin: 10px 0 50px 0
+
+.topic
+    padding: 20px 0
```

Comment:
To center something, it's better to use display:flex, rather then paddings/margins